### DISCUSSION TITLE
Crown allometry calibration

### DISCUSSION THREAD

**Discussion by rosiealice** [NGEET/fates#941] [General]
I came across this paper just now:

https://www.researchgate.net/profile/Vladimir_Usoltsev2/publication/304990965_Allometric_equations_for_integrating_remote_sensing_imagery_into_forest_monitoring_programmes/links/58afad64a6fdcc6f03f358d1/Allometric-equations-for-integrating-remote-sensing-imagery-into-forest-monitoring-programmes.pdf

Figure 4c in particular has some PFT-specific terms for crown allometry parameterizations that we should perhaps think about making sure we have some consistancy with.

**Comment by ckoven** [comment]
Let's use this discussion thread for allometry calibration issues.  Cross-linking some old issues that also focused on allometries: https://github.com/NGEET/fates/issues/619 and https://github.com/NGEET/fates/issues/352

**Comment by niuhanlin** [comment]
I have done some sensitivity tests for allometric growth of alpine grasslands on the Tibetan Plateau, but since it is a single point I do not know whether it is representative.
And when I changed the allometric parameters, the changes in the seed bank exceeded my expectations.
So, if there is a good solution, what can I do?
